src/vm/gcheaphashtable.h

```diff
@@ -0,0 +1,144 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#ifndef GCHEAPHASHTABLE_H
+#define GCHEAPHASHTABLE_H
+
+class GCHeapHashObject;
+
+template <bool remove_supported>
+struct DefaultGCHeapHashTraits
+{
+    typedef PTRARRAYREF THashArrayType;
+    static const INT32 s_growth_factor_numerator = 3;
+    static const INT32 s_growth_factor_denominator = 2;
+
+    static const INT32 s_density_factor_numerator = 3;
+    static const INT32 s_density_factor_denominator = 4;
+
+    static const INT32 s_densitywithdeletes_factor_numerator = 7;
+    static const INT32 s_densitywithdeletes_factor_denominator = 8;
+
+    static const INT32 s_minimum_allocation = 7;
+
+    static const bool s_remove_supported = remove_supported;
```

Comment:
It seems this constant is not needed as it is not accessed from the outside, so we can use the template parameter directly.

Reply:
Removed